include/ecs/entity.h

```diff
@@ -0,0 +1,18 @@
+#ifndef ECS_ENTITY_H
+#define ECS_ENTITY_H
+
+#include <bitset>
+#include "common.h"
+
+namespace arch::ecs {
+	class Entity {

```

Comment:
If all you need is public properties and constructor, you should use struct.

Reply:
Done